pkgs/top-level/all-packages.nix

```diff
@@ -94,6 +94,8 @@ in
 
   genericUpdater = callPackage ../common-updater/generic-updater.nix { };
 
+  unstableUpdater = callPackage ../common-updater/unstable-updater.nix { };
```

Comment:
We probably should make the name signal that it is git specific.

Reply:
I based the name on genericUpdater. Should I rename that one too or is it too late?

Reply:
`genericUpdater` is quite fresh an undocumented so I do not see an issue with renaming it. We can leave it for a separate PR, though, if we decide to not use it now.

Reply:
Done

Reply:
I named it "unstableGitUpdater", not sure if it's clear enough